### PR TITLE
New Package fparser@2015/09/25

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

**Review by baberlevi — APPROVED** [review]

File: var/spack/repos/builtin/packages/fparser/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fparser(CMakePackage):
+    """This C++ library offers a class which can be used to parse and
+    evaluate a mathematical function from a string (which might be eg
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/fparser/package.py]: Can we have a short one-line description first and a longer description as a separate paragraph:
```suggestion
    """Parse and evaluate a mathematical function from a string.
    
    This C++ library offers a class which can be used to parse and
    evaluate a mathematical function from a string (which might be eg
```
?

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a minor request

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
There's a flake8 issue, sorry:
```
var/spack/repos/builtin/packages/fparser/package.py:11: [W293] blank line contains whitespace
```
Ping me when you solved that and I'll merge.

**Comment by dorton21** [comment]
In hindsight this isn't needed and it is a minor package.

**Pull request closed without merging**